src/graphorm/index.ts

```diff
@@ -242,6 +247,7 @@ const obj = new GraphORM({
             return null;
           }
 
+          // TODO member-invite-rank pass memberInviteRank to resolve Invite permission for member
           return getPermissionsForMember(member, { memberPostingRank });
```

Comment:
@rebelchris remember when we were adding above `select` to get `memberPostingRank` to then filter permissions array to not include `Post` permission depending on the `memberPostingRank` column.

I tried to add the same thing for `memberInviteRank` (already implemented it inside `getPermissionsForMember`) but when I try to add `memberInviteRank` column to the above `select` I get the error `error: subquery must return only one column`. Not sure what alternatives we have, tried with customRelation but it did not work same as when we tried before.

Reply:
Hacky way came to mind to use `concat("memberPostingRank", ':', "memberInviteRank")` in select and then in transform `const [memberPostingRank, memberInviteRank] = value.split(':')`. It works but I would rather find alternatives. This basically just cheats psql to think single column was returned.

Reply:
I think it's because the :string value, you should be able to expect an array I think.
Let me know if you want to try and debug it together quickly?

Reply:
@idoshamun could you take a look at this one since we could not find the solution? Basically the issue is that we want to query two two columns from `select` in graphorm but can not due to error above.

Reply:
`array[colum1, column2]` works and it also returns native types eg. for our case `[number, number]`.